core/config/engine.h

```diff
@@ -75,6 +81,8 @@ class Engine {
 	List<Singleton> singletons;
 	HashMap<StringName, Object *> singleton_ptrs;
 
+	Vector<ExtensionClassDoc> extensions_class_doc;
```

Comment:
Could also be `List` or `LocalVector`. As it's passed around via a getter I figured `Vector` might be correct but I'm no expert here.